File: src/main/java/no/nav/pto/veilarbportefolje/skjerming/SkjermingData.java
package no.nav.pto.veilarbportefolje.skjerming;

import no.nav.common.types.identer.Fnr;

import java.sql.Timestamp;

public record SkjermingData(Fnr fnr, boolean er_skjermet, Timestamp skjermet_fra, Timestamp skjermet_til) {
}

File: src/main/java/no/nav/pto/veilarbportefolje/skjerming/SkjermingData.kt
package no.nav.pto.veilarbportefolje.skjerming

import no.nav.common.types.identer.Fnr
import java.sql.Timestamp

data class SkjermingData(
    val fnr: Fnr,
    val erSkjermet: Boolean,
    val skjermetFra: Timestamp?,
    val skjermetTil: Timestamp?
)

